source/app/pages/documents/view.js

```diff
@@ -383,9 +394,12 @@ function Document({ currentPageNumber, dispatch, id, document, pageTitle, search
             />
 
             <div
-              className={cs(
+              className={ENABLE_COMPREHEND_MEDICAL? cs(
```

Comment:
Why do we need this? Isn't it impossible for tab === 'medical_entities' to be true if the feature is disabled anyway?

Reply:
yeah I figured that later